application.tf

```diff
@@ -1093,14 +1107,10 @@ crunchy-postgres:
 ingress:
   useStaticCertificate: false
   secretName: indico-ssl-static-cert
-  tls.crt: #base64 encoded value of certificate chain
-  tls.key: #base64 encoded value of certificate key
 minio:
-  topology:
-    volumeSize: 128Gi
   storage:
-    accessKey: <path:tools/argo/data/indico-dev/ins-dev/storage#access_key_id>
-    secretKey: <path:tools/argo/data/indico-dev/ins-dev/storage#secret_access_key>
+    accessKey: ${random_password.minio-username.result}
```

Comment:
Looks like the secrets being referenced here are AWS credentials to connect to a storage bucket. Is it okay to set these as minio:<random_password>?

Have you tested deploying a cluster with this change?

Reply:
they're configure the root username & password for minio, not to an external s3 iam user